src/actions/traderAction.js

```diff
@@ -142,3 +142,13 @@ export const fetchTradersByDate = (startDate, endDate, token) => async dispatch
     console.log(err);
   }
 };
+
+export async function convertToken(token) {
+  const cors = 'https://cors-anywhere.herokuapp.com/';
```

Comment:
I have a concern about the use of this - i would prefer to use our own proxy rather than rely on this 3rd party app. Please talk to @adrian-gierakowski about this as he can provide an example

Reply:
Ok, i will replace it. Could you just provide me this url?

Reply:
I will merge for now and create an issue to replace later